Form/Extension/DependencyInjectionExtension.php

```diff
@@ -38,29 +44,31 @@ class DependencyInjectionExtension implements FormExtensionInterface
     private $container;
 
     /**
-     * @var []string
+     * @var array
```

Comment:
1. I meant I'm fine with how it was before
2. Actually I'm not fine, it should be `string[]`